PreFreeSurfer/scripts/BiasFieldCorrection_sqrtT1wXT1w.sh

```diff
@@ -99,19 +102,23 @@ echo " " >> $WD/log.txt
 ########################################## DO WORK ########################################## 
 
 # Form sqrt(T1w*T2w), mask this and normalise by the mean
+verbose_echo " --> Forming rt(T1w*T2w), masking this and normalising by the mean"
```

Comment:
This should say " --> Forming sqrt(T1w*T2w), ..."

Reply:
done.